webapp/channels/src/utils/utils.tsx

```diff
@@ -1286,7 +1286,7 @@ export async function handleFormattedTextClick(e: React.MouseEvent, currentRelat
                             let post = getPost(state, postId!);
                             if (!post) {
                                 const {data: postData} = await store.dispatch(getPostAction(match.postId!));
-                                post = postData;
+                                post = postData!;
```

Comment:
Does this always return a potentially `undefined` value? Maybe the return type of `getPostAction` is wrong?

Reply:
The changes in this function match the current behaviour at least. I do agree that these should actually handle if an error is thrown, but this would behave the same as before at least.

Reply:
Yes, I prefer focus in adding types, later we can look for all the "!" in the code and double check them.